src/main/kotlin/net/atos/zac/util/BSNValidator.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.util
+
+object BSNValidator {
+
+    private const val BSN_LENGTH = 9
+    private const val BSN_MULTIPLE = 11
```

Comment:
Maybe rename this const to make it more clear? It's hard to understand what 'BSN multiple' means.. Probably good to add that link to the BSN 11-proof web page here as well.

Reply:
Added comment and renamed to BSN_11_PROEF